manifests/r/RaynerSec/Hyper-V-Switch/1.0/RaynerSec.Hyper-V-Switch.locale.en-US.yaml

```
@@ -0,0 +1,30 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: RaynerSec.Hyper-V-Switch
+PackageVersion: 1.0
+PackageLocale: en-US
+Publisher: RaynerSec
+PublisherUrl: https://github.com/RaynerSec
+PublisherSupportUrl: https://github.com/RaynerSec/Hyper-V-Switch/issues
+Author: RC Chuah
+PackageName: Hyper-V-Switch
+PackageUrl: https://github.com/RaynerSec/Hyper-V-Switch
+License: GNU General Public License v3.0
+LicenseUrl: https://github.com/RaynerSec/Hyper-V-Switch/blob/main/LICENSE
+Copyright: Copyright (C) 2022 RaynerSec. Licensed under the GNU GPL v3.0.
+CopyrightUrl: https://github.com/RaynerSec/Hyper-V-Switch/blob/main/LICENSE
+ShortDescription: This Program Enables And Disables Hyper-V Hypervisor So You Can Use Other Virtualisation Tools Such As (VMware, VirtualBox) Simultaneously.
+Description: This Program Enables And Disables Hyper-V Hypervisor So You Can Use Other Virtualisation Tools Such As (VMware, VirtualBox) Simultaneously.
+Moniker: hyper-v-switch
+Tags:
+- Hyper-V
+- hyper-v
+- Hyper-V-Switch
+- hyper-v-switch
+- WSL
+- wsl

```

```suggestion
- hyper-v
- hyper-v-switch
- wsl
```
Tags should be lowercased, and not duplicated